Remotion/ObjectBinding/Web.Test/Experimental/ControlLayout-v1.htm

```diff
@@ -92,7 +93,8 @@
       box-sizing: border-box;
       -moz-box-sizing: border-box;
       -webkit-box-sizing: border-box;
-      background-image: url('../res/Remotion.ObjectBinding.Web/Themes/NovaBlue/Image/AutoCompleteReferenceValueDropDownButtonFace.gif');
+      background-image: url('../res/Remotion.ObjectBinding.Web/Themes/NovaBlue/Image/sprite.svg#AutoCompleteReferenceValueDropDownButtonFace');
```

Comment:
I'm sorry, I just realized I missed something in the review of the SVG conversion:
* AutoCompleteReferenceValueDropDownButtonFace
* DropDownMenuButton
are already sprites, so they need to be broken up into individual images before they can be integrated into a sprite-svg.

And 
* DropDownMenuRequestInProgressIndicator
* AutoCompleteReferenceValueRequestInProgressIndicator
are animated GIFs. I don't think they still work

Please fix either as a bugfix-task before this PR or at the start of this PR.

If the animated GIFs don't work in SVG, we probably have to keep them as GIF for now, and get back to this later -> follow-Up ticket.

Reply:
okay, sorry, update:
It looks like the svg-background actually works with the merged sprites and the dependencies on them are only inside the CSS, we don't need to change the files for the old theme. I mean, we could, as a separate follow-up Task, but we don't have to.

We still need to fix the animated GIFs, though. It does seem that they work in Chrome/Edge but not in Firefox.

Reply:
Bugreport: I meant create a re-motion ticket for this with a caused-by for the current task.

Reply:
Added https://re-motion.atlassian.net/browse/RM-7972